### ISSUE TITLE
Cannot GetActorsAsync in Azure cluster

### ISSUE THREAD

**Issue by ddimitrov90** [Azure/service-fabric-services-and-actors-dotnet#109]
Hello,

I am trying to verify that an actor in a stateful persisted actor service has been created in our integration tests. I am using the following snippet:
```csharp
ServicePartitionList partitionList = await this.FabricClient.QueryManager.GetPartitionListAsync(new Uri($"{actor-service-name}"));
List<string> actorIds = new List<string>();

foreach (var partition in partitionList)
{
	var key = partition.PartitionInformation as Int64RangePartitionInformation;
	if (key != null)
	{
		IActorService actorServiceProxy = ActorServiceProxy.Create(new Uri($"{actor-service-name}"), key.LowKey);
		ContinuationToken continuationToken = null;
		var cancellationTokenSource = new CancellationTokenSource();
		do
		{
			PagedResult<ActorInformation> page = await actorServiceProxy.GetActorsAsync(continuationToken, cancellationTokenSource.Token);
			actorIds.AddRange(page.Items.Select(x => (string)x.ActorId.GetStringId()));

			continuationToken = page.ContinuationToken;
		}
		while (continuationToken != null);
	}
}
```

This code works without problems on the local environment, however when the cluster is deployed in Azure I receive an exception ( the cluster is with the default setup/firewall/load balancing rules):
1. When the test is ran from VSTS, the exception is:
```
System.AggregateException : One or more errors occurred.
----> System.Fabric.FabricInvalidAddressException : The supplied address was invalid.
----> System.Runtime.InteropServices.COMException : Exception from HRESULT: 0x80071BBD
```

2. I ran the test locally from my machine, with dlls from this repository build in Debug and the problem occurs in ServicePartitionResolver -> ResolveInt64PartitionAsync with a ServiceNotFound Exception. 
At the time of debugging, I have connected to the cluster in powershell and Get-ServiceFabricServiceDescription and Get-ServiceFabricServiceDescription commands are giving me the correct result, so the service is up and running.

It would be great if you can share some info what might be closed/blocked/firewall/etc... in the cloud setup.

**Comment by amanbha** [comment]
@ddimitrov90 You have to be within the cluster to make any Actor/Service remoting call.

**Comment by ddimitrov90** [comment]
@amanbha the fabricClient is of course connected to the cluster in the cloud.

i can successfully get the partitions list through the querymanager, but when i call the GetActorsAsync, the above mentioned error occurs

**Comment by amanbha** [comment]
@ddimitrov90 yes, I understand that. For making actor calls or service remoting calls, the client should be in the cluster.
If you want to query from outside, you will have to add some gateway(it can be an aspnetcore app) in the cluster which can accpet call from outside and can make actor calls on behalf of incoming request.
